modules/coreos-layering-removing.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * post-installation_configuration/coreos-layering.adoc
+
+:_content-type: PROCEDURE
+[id="coreos-layering-removing_{context}"]
+= Removing a {op-system} custom layered image
+
+You can easily revert {op-system-first} image layering from the nodes in specific machine config pools. The Machine Config Operator (MCO) reboots those nodes with the cluster base {op-system-first} image, overriding the custom layered image.
+
+To remove a {op-system-first} custom layered image from your cluster, you need to delete the machine config that applied the image.
+
+.Procedure
+
+. Delete the machine config file that applied the custom layered image.
++
+[source,terminal]
+----
+$ oc delete mc rendered-worker-274fe73fca83304164c636f28a74525f
```

Comment:
We should delete the MachineConfig that configures the new osImage, not the generated rendered machine config . The user should never delete an automatically  rendered machine config.

Using the previous example in the documentation, the command should delete os-layer-hotfix instead of rendered-worker-274fe73fca83304164c636f28a74525f. Something like this:

```
oc delete mc os-layer-hotfix 
```